kotlin-ring-ui/src/main/kotlin/ringui/Loader.ext.kt

```diff
@@ -1,7 +1,7 @@
 package ringui
 
-var LoaderProps.dataTest: String
-    get() = asDynamic()["data-test"] as? String ?: ""
+var LoaderProps.dataTest: String?
+    get() = asDynamic()["data-test"] as String?
```

Comment:
Cast is redundant

Reply:
IntelliJ Idea gives following warning if remove cast: "Implicit (unsafe) cast from dynamic to String? " :/

Reply:
But it has no big sense, because you emulate external property
Otherwise you will lock smooth migration to property